### PR TITLE
Return group id and name with phylo tree

### PR DESCRIPTION
**Pull request by lvreynoso**
### Summary:
- **What:** Returns a `group` key with each phylo tree entry, an object with the group's id and name.
- **Ticket:** [sc177022](https://app.shortcut.com/genepi/story/177022)
- **Env:** `<rdev link>`

### Demos:
Screen Shot 2021-12-15 at 11 16 10

### Checklist:
- [x] I merged latest `<base branch>`
- [x] I manually verified the change
- [x] I added labels to my PR
- [x] I tested in multiple browsers
- [x] I added relevant unit tests
- [x] I have notified others of changes they need to make locally (migrations, jobs, package updates, etc)

### CODE REVIEW

**Review by mmmmmmaya — APPROVED** [review]
Thank you!

File: src/backend/aspen/app/views/phylo_trees.py [hunk 1]
```diff
@@ -95,6 +96,10 @@ def phylo_trees():
             "pathogen_genome_count": 0,  # TODO: do we still need this?,
             "tree_type": phylo_run.tree_type.value,
             "user": {},
+            "group": {
+                "id": phylo_run.group.id,
+                "name": phylo_run.group.name,
+            },
```
Comment (jgadling) [inline src/backend/aspen/app/views/phylo_trees.py:102]: Are we still calling the old phylo trees endpoint?

Comment (lvreynoso) [reply src/backend/aspen/app/views/phylo_trees.py:102]: Yup. We will have to pick back up the migration when we can

Comment (lvreynoso) [reply src/backend/aspen/app/views/phylo_trees.py:102]: Oh yeah... @mmmmmmaya the v2 endpoint returns this information already, but the information is formatted differently. Do you want to just use the new endpoint?

**Comment by mmmmmmaya** [comment]
It will require non trivial changes to the app to use the new endpoint. I can make a ticket to use the v2 endpoint in the future, but in the interest of finishing crud v0 before end of year, I'd like to use the existing endpoint.

**Pull request merged**